doc/api/fs.md

```diff
@@ -2805,7 +2815,7 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `flags` {string|number} See [support of file system `flags`][].
+* `flag` {string|number} See [support of file system `flags`][].
```

Comment:
To be fair, the FS API and documentation already uses those inconsistently (E.G.: `flags` in [`fs.openSync`](https://nodejs.org/api/fs.html#fs_fs_opensync_path_flags_mode), `flag` in [`fs.readFileSync`](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options)).

Reply:
Note that changing the documentation from `flags` to `flag` would break the existing hypertext links (from StackOverflow, blog posts, etc.), which is something we'd rather not do.